### ISSUE TITLE
Fullscreen video is missing

### ISSUE THREAD

**Issue by gastongaston** [steventroughtonsmith/tvOSBrowser#11]
Hi, I like the app, it is great. However, one important thing is missing, fullscreen mode for videos. If this was added to the app it would be improved by a lot. I think the youporn viewer app ( https://github.com/kimiko88/Tvos-YouPornViewer ) has implemented fullscreen quite nicely.

Thanks in advance

**Comment by tomsbrinkis** [comment]
how about this? :)

**Comment by WyattMufson** [comment]
Full screen still not working.

**Comment by WyattMufson** [comment]
Full screen still doesn't work.

**Comment by tomsbrinkis** [comment]
Yep. And i Think, that this will not be changed. The project is old. Maybe we need to discuss thiss in Reddit?
